core/src/main/resources/hudson/widgets/HistoryWidget/index.jelly

```diff
@@ -79,16 +79,12 @@ THE SOFTWARE.
             <input type="search" placeholder="${%find}" class="jenkins-search__input" />
             <l:svgIcon href="${resURL}/images/material-icons/svg-sprite-action-symbol.svg#ic_search_24px" class="jenkins-search__icon" />
           </div>
-          <j:if test="${page.runs.isEmpty()}">
-            <div id="no-builds" class="jenkins-pane__information">
-              No builds
-            </div>
-          </j:if>
-          <j:if test="${!page.runs.isEmpty()}">
-            <div id="no-builds" class="jenkins-pane__information" style="display: none;">
-              No builds
-            </div>
-          </j:if>
+          <x:element name="div">
+            <x:attribute name="id">no-builds</x:attribute>
+            <x:attribute name="class">jenkins-pane__information</x:attribute>
+            <x:attribute name="style">${page.runs.isEmpty() ? "" : "display: none;"}</x:attribute>
+            ${%No builds}
+          </x:element>
```

Comment:
why not just (untested):
(doesn't matter if there's an empty style tag)
```suggestion
          <div id="no-builds" class="jenkins-pane__information" style="${page.runs.isEmpty() ? "" : "display: none;"}">
            ${%No builds}
          </div>
```

Reply:
This makes a ton more sense - not too sure what was going through my head 🙈 Thanks :)